### ISSUE TITLE
After login - no tables visible when created through gui

### ISSUE THREAD

**Issue by yure** [interchange/TableEditor#71]
After entering connection data through gui, you see "tables" and "settings" on the left but there is no content. test database is mysql with 1 table

**Comment by yure** [comment]
mysql> describe cdr; 
+-------------+--------------+------+-----+---------------------+-------+
| Field       | Type         | Null | Key | Default             | Extra |
+-------------+--------------+------+-----+---------------------+-------+
| calldate    | datetime     | NO   |     | 0000-00-00 00:00:00 |       | 
| clid        | varchar(80)  | NO   |     |                     |       | 
| src         | varchar(80)  | NO   |     |                     |       | 
| dst         | varchar(80)  | NO   |     |                     |       | 
| dcontext    | varchar(80)  | NO   |     |                     |       | 
| channel     | varchar(80)  | NO   |     |                     |       | 
| dstchannel  | varchar(80)  | NO   |     |                     |       | 
| lastapp     | varchar(80)  | NO   |     |                     |       | 
| lastdata    | varchar(80)  | NO   |     |                     |       | 
| duration    | int(11)      | NO   |     | 0                   |       | 
| billsec     | int(11)      | NO   |     | 0                   |       | 
| disposition | varchar(45)  | NO   |     |                     |       | 
| amaflags    | int(11)      | NO   |     | 0                   |       | 
| accountcode | varchar(20)  | NO   |     |                     |       | 
| uniqueid    | varchar(32)  | NO   | MUL |                     |       | 
| userfield   | varchar(255) | NO   |     |                     |       | 
+-------------+--------------+------+-----+---------------------+-------+
16 rows in set (0.01 sec)

**Comment by racke** [comment]
I suggest to remove default password from config and give admin full permissions by default.

**Comment by gregapompe** [comment]
Agreed. You probably don't have permission, but of course you are unaware. I'll solve it.

**Comment by gregapompe** [comment]
I removed admin pass and added default permissions. Can you please test it yure?

**Comment by yure** [comment]
it says:


    Connection to DB

Connected successfully to *mysql* database named *asteriskcdrdb* with 
user *cdr*!


but still no tables on the left side.

(i just did the pull).

also still can't do pull from GUI
